### PR TITLE
DOCSP-42321: Insert landing page

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42321>
Staging - <https://deploy-preview-55--docs-scala.netlify.app/write/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write.txt [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+   /write/update
+   /write/delete
+   /write/bulk-write
+   /write/transactions
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common methods you
+can use to write data to MongoDB with the {+driver-short+}.
```
Comment (rustagir) [inline source/write.txt:36]:
```suggestion
can use to write data to MongoDB by using the {+driver-short+}.
```

File: source/includes/usage-examples/WriteCodeExamples.scala [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    observable.subscribe(new Observer[UpdateResult] {
+      override def onNext(result: UpdateResult): Unit = println(result)
+      override def onError(e: Throwable): Unit = println("Failed: " + e.getMessage)
+      override def onComplete(): Unit = println("Completed")
+    })
+    // end-update-many
+
+    // start-replace-one
+    val replacementDoc: Document = Document("<field name>" -> "<value>")
+    val filter = equal("<field to match>", "<value to match>")
```
Comment (rustagir) [inline source/includes/usage-examples/WriteCodeExamples.scala:67]: S: reverse the order of these two lines

**Review by rustagir — APPROVED** [review]
lgtm with a few small improvements!

File: source/write.txt (hunk 1 above)
Comment (mcmorisi) [reply source/write.txt:36]: Every time, this one gets me!

File: source/includes/usage-examples/SampleWriteApp.scala [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+    val database: MongoDatabase = mongoClient.getDatabase("<database name>")
+    val collection: MongoCollection[Document] = database.getCollection("<collection name>")
+
+    // Start example code here
+
+    // End example code here
+
+    // Wait for the operations to complete before closing client
```
Comment (rozza) [inline source/includes/usage-examples/SampleWriteApp.scala:20]:
```suggestion
    // Wait for the operations to complete before closing client
    // Note: This example uses `Thread.sleep` for brevity 
    // this does not guarantee all operations will be completed in time.
```

I think its important to note to users the sleep provides no guarantees.

**Review by rozza — CHANGES_REQUESTED** [review]

**Review by rozza — APPROVED** [review]
LGTM

**Pull request merged**